### ISSUE TITLE
Wanted: Config option for bigger font in crafting preview

### ISSUE THREAD

**Issue by Knito58** [raoulvdberge/refinedstorage#1713]
The config option in "grid" for bigger font just sets a real ugly big font into the grid. But the crafting preview font still remains very small and is unreadable as at least one row of pixels seems to be missing. 

I am one of these older people with not-so-good-eyes who need readable infos .... Even with glasses the writing is a guessing game.

**Comment by raoulvdberge** [comment]
I see, but I'm pretty sure a big font in the crafting preview window wouldn't really fit, size-wise.

You might want to turn on the unicode font in the language settings to get a bigger font.

**Comment by raoulvdberge** [comment]
You could also try increasing the GUI size in the settings menu.

Can you report back your findings?

**Comment by Knito58** [comment]
Ok I tried different GUI sizes. "Small" is too small for me. "Large" is readable and "auto" even better. 
But "large" and "auto" both increase the size of the GUI. On my 1280 x 1024 screen the overpowered inventory only shows the center and a part of the hotbar when "large" or "auto" is used.

Setting the font to unicode makes a very thin font which is too fine at first impression. But it solves my problem with the writing in the crafting preview. Although the messages do not respect the boxes sizes they are clearly readable. 

I found my new settings: GUI normal and unicode ON. I think, I will get used to it.
Thank you.
